ValveAnnulusAnalysis/HeartValveWidgets/ValveSequenceBrowserWidget.py

```diff
@@ -0,0 +1,267 @@
+import logging
+from string import Template
+from enum import IntEnum
+
+import vtk
+import slicer
+from slicer.util import VTKObservationMixin
+
+import HeartValveLib
+from HeartValveLib.Constants import CARDIAC_CYCLE_PHASE_PRESETS
+
+
+MODULE_NAME = "ValveAnnulusAnalysis"
+
+
+class Permission(IntEnum):
```

Comment:
If we foresee multiple permissions (add/remove time points, valve type, valve volume, probe position) then we could make it a [Flag](https://docs.python.org/3/library/enum.html#flag) so that these permissions can be enabled independently. If we envision only read-only Boolean then the Qt convention is to have a `readOnly` Boolean flag.